pkg/collector/ecr_events.go

```diff
@@ -387,7 +414,7 @@ func getVulnerabilitySeverity(v string) vulnerability.Severity {
 	}
 }
 
-func (i *ecrCollector) newImageScanOccurrences(event *CloudWatchEvent, detail *ECRImageScanDetail) []*grafeas.Occurrence {
+func (i *ecrCollector) newImageScanOccurrences(event *CloudWatchEvent, detail *ECRImageScanDetail) (occurrences []*grafeas.Occurrence, err error) {
```

Comment:
nitpick: we aren't using named outputs in most of the codebase, can these be changed to unnamed outputs?

Reply:
Done 👍 , the original reason for this was to prevent having to declare an error variable to consume the error value returned from `getVulnerabilityDetails` since I didn't like the look of it. But also didn't like making this the only function with named outputs so I could've gone either way 🤷 .